libbeat/autodiscover/providers/kubernetes/pod.go

```diff
@@ -404,7 +405,8 @@ func (p *pod) containerPodEvents(flag string, pod *kubernetes.Pod, c *containerI
 			flag:         true,
 			"kubernetes": kubemeta,
 			// Actual metadata that will enrich the event.
-			"meta": meta,
+			"meta":         meta,
+			"orchestrator": orchestrator,
```

Comment:
I think that to actually enrich the events, the new fields should be set in `meta`. I think that the other fields are used only in autodiscover (for templates, conditions and so on) but don't end up in the final documents. (But not 100% sure, I always get confused by that).